### PR TITLE
added calendar

### PR DESCRIPTION
**Pull request by khushigarg1**
#145 by[khushigarg1]
img

### CODE REVIEW

File: Projects/Calendar/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<!-- Coding By CodingNepal - youtube.com/codingnepal -->
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8" />
+    <title>Dynamic Calendar JavaScript | CodingNepal</title>
+    <link rel="stylesheet" href="styles.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Google Font Link for Icons -->
+    <link
```
Comment (LiQuiD-404) [inline Projects/Calendar/index.html:10]: @khushigarg1 Could you explain this? This project seems to have be copied.
Link : https://www.youtube.com/watch?v=Z1BGAivZRlE

Comment (khushigarg1) [reply Projects/Calendar/index.html:10]: sorry i just took a html file from there but css and all and some animations that i added myself.

Comment (LiQuiD-404) [reply Projects/Calendar/index.html:10]: @khushigarg1 It's okay to refer to YouTube videos for your projects but make sure you make the final project on your own. Best of Luck ✨

Comment (khushigarg1) [reply Projects/Calendar/index.html:10]: > @khushigarg1 Could you explain this? This project seems to be copied from the below mentioned youtube channel. Link : https://www.youtube.com/watch?v=Z1BGAivZRlE

Yes thank you!

File: Projects/Calendar/index.html [hunk 2]
```diff
@@ -1,12 +1,10 @@
 <!DOCTYPE html>
-<!-- Coding By CodingNepal - youtube.com/codingnepal -->
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8" />
     <title>Dynamic Calendar JavaScript | CodingNepal</title>
```
Comment (Vanshiii203) [inline Projects/Calendar/index.html:5]: hey @khushigarg1 Change the title of the project

Comment (khushigarg1) [reply Projects/Calendar/index.html]: Done!

**Review by harshit-paneri — CHANGES_REQUESTED** [review]
Add some new feature and send it again

**Comment by harsh5902** [comment]
great @khushigarg1 👍

**Review by harsh5902 — APPROVED** [review]

**Comment by harsh5902** [comment]
Try to add more features to this project like remainder of a day or saving some text for some day and many more like that.

**Review by TusharKesarwani — APPROVED** [review]

**Pull request merged**